### PR TITLE
onefetch: init at 1.7.0

### PR DESCRIPTION
**Pull request by kloenk**
###### Motivation for this change
Adds onefetch for git repos analysis

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3484,6 +3484,16 @@
     github = "klntsky";
     githubId = 18447310;
   };
+  kloenk = {
```
Comment (mmahut) [inline maintainers/maintainer-list.nix]: Do you mind creating a separate commit for adding an user?

File: pkgs/tools/misc/onefetch/default.nix [hunk 2]
```diff
@@ -0,0 +1,35 @@
+{
+    cargo
+  , makeRustPlatform
+  , fetchFromGitHub
+  , lib
+  , ...
```
Comment (mmahut) [inline pkgs/tools/misc/onefetch/default.nix]: What is the purpose?

File: pkgs/tools/misc/onefetch/default.nix [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      rev = version;
+      sha256 = "1p16mg4ak9ppx3y11l3r4y6356drwhnmrlxsaqx01n53ii5ij9kg";
+    };
+    cargoSha256 = "0cpfjxn6nqsrnmgvjzr17n6xbbyfl9c91d5kbrmxb6jyig98k6aq";
+    buildInputs = [ ];
+    CARGO_HOME = "$(mktemp -d cargo-home.XXX)";
+
+    meta = with lib; {
+      homepage = https://github.com/o2sh/onefetch;
+      description = ''Displays information about your Git project directly on your terminal'';
```
Comment (mmahut) [inline pkgs/tools/misc/onefetch/default.nix]: Please don't use ``''`` if no multi line.

File: pkgs/tools/misc/onefetch/default.nix [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      repo = "onefetch";
+      rev = version;
+      sha256 = "1p16mg4ak9ppx3y11l3r4y6356drwhnmrlxsaqx01n53ii5ij9kg";
+    };
+    cargoSha256 = "0cpfjxn6nqsrnmgvjzr17n6xbbyfl9c91d5kbrmxb6jyig98k6aq";
+    buildInputs = [ ];
+    CARGO_HOME = "$(mktemp -d cargo-home.XXX)";
+
+    meta = with lib; {
+      homepage = https://github.com/o2sh/onefetch;
```
Comment (mmahut) [inline pkgs/tools/misc/onefetch/default.nix]: Please use quotes if possible.

File: pkgs/tools/misc/onefetch/default.nix [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    
+    src = fetchFromGitHub {
+      owner = "o2sh";
+      repo = "onefetch";
+      rev = version;
+      sha256 = "1p16mg4ak9ppx3y11l3r4y6356drwhnmrlxsaqx01n53ii5ij9kg";
+    };
+    cargoSha256 = "0cpfjxn6nqsrnmgvjzr17n6xbbyfl9c91d5kbrmxb6jyig98k6aq";
+    buildInputs = [ ];
+    CARGO_HOME = "$(mktemp -d cargo-home.XXX)";
```
Comment (mmahut) [inline pkgs/tools/misc/onefetch/default.nix]: Isn't this created by fetchCargo?

File: pkgs/tools/misc/onefetch/default.nix [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    version = "1.7.0";
+    
+    src = fetchFromGitHub {
+      owner = "o2sh";
+      repo = "onefetch";
+      rev = version;
+      sha256 = "1p16mg4ak9ppx3y11l3r4y6356drwhnmrlxsaqx01n53ii5ij9kg";
+    };
+    cargoSha256 = "0cpfjxn6nqsrnmgvjzr17n6xbbyfl9c91d5kbrmxb6jyig98k6aq";
+    buildInputs = [ ];
```
Comment (mmahut) [inline pkgs/tools/misc/onefetch/default.nix]: What is the reason to have an empty buldInputs?

File: pkgs/tools/misc/onefetch/default.nix (hunk 2 above)
Comment (jonringer) [inline pkgs/tools/misc/onefetch/default.nix]: not needed
```suggestion
```

**Comment by Kloenk** [comment]
I hade not so much time, because of school, so I will close this PR in favor of #74191
